app/js/calculator.js

```diff
@@ -158,6 +158,9 @@ var Calculator = {
   handleEvent: function handleEvent(evt) {
     var target = evt.target;
     var value = target.value;
+
+    // Adding haptic feedback on key press
+    window.navigator.vibrate(50);
```

Comment:
looks good, we've had vibration support since well before 18 so this shouldn't break anything. Can you please remove `window.`?